.idea/inspectionProfiles/profiles_settings.xml

```diff
@@ -0,0 +1,6 @@
+<component name="InspectionProjectProfileManager">
```

Comment:
remove meta files or add those into gitignore

Reply:
How do I do it?

Reply:
take a look here https://www.w3schools.com/git/git_ignore.asp